flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/crd/status/JobStatus.java

```diff
@@ -30,7 +32,7 @@
 public class JobStatus {
     private String jobName;
     private String jobId;
-    private String state;
```

Comment:
Why are we renaming this and not other fields? I agree that it is not super consistent we 
jobName, jobId, state, updateTime, savepointLocation

Wouldnt it be better to have simply name, id, state ...

Reply:
Maybe this could be a separate JIRA discussion to improve field naming in the spec/status and do multiple changes together to simplify testing (redeploying CRD etc.)

Reply:
So to summarize, my suggestion is to not change the name of the field in this PR, but instead let's open a ticket where we gather some improvements to the spec/status instead.

Reply:
Sure. It makes sense to me to leave the name change to another ticket. The renaming has been rolled back.
The reason of renaming here is to follow the naming convention of `org.apache.flink.runtime.client.JobStatusMessage` of flink. We could discuss if such change is necessary later.